site/config.toml

```diff
@@ -0,0 +1,13 @@
+languageCode = "en-us"
+title = "NuKeeper"
+theme = "hugo-theme-learn"
+
```

Comment:
```suggestion
enableEmoji = true
```

Reply:
This will show the emoji checkmarks where we use them

Reply:
wow we definitely need that!

Reply:
done 👍 ⚡️